airship/kubeadm-airship-gap-analysis.md

```diff
@@ -13,6 +13,59 @@ Jira Issues:
 Jira Issues:
 - [CA rotation](https://airship.atlassian.net/browse/AIR-138)
 
+ 
+## Use pre-generated certs and user provided CA
+
+This is possible with Kubeadm. We can provide CA certs only or we can provide the whole list of certificates in ```/etc/kubernetes/pki``` . In case we provide CA cert only, kubeadm will generate the other certificates using the CA cert. In the latter case, kubeadm will take the pre-generated certs and avoid generating new ones. The placement of the certs can be done using a script.
```

Comment:
change to

We can provide either only CAs or ....

Reply:
Done